README.md

```diff
@@ -175,7 +175,4 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 
-A copy of the license is available in the repository's [LICENSE](./license.txt) file.
-
-[](Esri Tags: ArcGIS Web Mapping Leaflet AppBuilder Hackerlabs Labs JavaScript)
-[](Esri Language: JavaScript)
+A copy of the license is available in the repository's [LICENSE](./LICENSE) file.
```

Comment:
I haven't checked, does this license hyperlink pattern work? It's referencing a "LICENSE" file in all caps and the license file itself is all lower case.